pkg/evpn/tunnel_vxlan.go

```diff
@@ -77,7 +77,7 @@ func (s *Server) CreateTunnel(_ context.Context, in *pb.CreateTunnelRequest) (*p
 			return nil, err
 		}
 		// now get Subnet/Bridge to plug this vxlan into
-		bridge, ok := s.Subnets[in.Tunnel.Spec.VpcNameRef]
+		bridge, ok := s.Bridges[in.Tunnel.Spec.VpcNameRef]
```

Comment:
VpcNameRef?

Reply:
ignore for now, will be refactored part of SVI and PORT changes